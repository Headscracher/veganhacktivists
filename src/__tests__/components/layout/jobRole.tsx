import React from 'react';
import { render } from '@testing-library/react';
import JobRole from '../../../components/layout/jobRole';
import codeLogo from '../../public/images/joinUs/VH-icon-dev.png';

it('should render correctly', () => {
  const { asFragment } = render(
    <JobRole image={codeLogo} color="red" title="foo" description="bar" />
  );
  expect(asFragment()).toMatchSnapshot();
});
