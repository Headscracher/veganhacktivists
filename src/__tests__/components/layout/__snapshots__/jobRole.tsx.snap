// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<DocumentFragment>
  <div
    class="bg-gray-background text-grey-dark"
  >
    <div
      class="relative overflow-visible hidden md:block opacity-30 "
    >
      <div
        class="absolute"
        style="background-color: rgb(22, 25, 25); width: 64px; height: 64px; opacity: 1; right: 0px;"
      />
    </div>
    <div
      class="md:h-52 flex flex-col justify-center py-8"
      style="background-color: rgb(237, 24, 74);"
    >
      <div
        style="display: inline-block; max-width: 100%; overflow: hidden; position: relative; box-sizing: border-box; margin: 0px;"
      >
        <div
          style="box-sizing: border-box; display: block; max-width: 100%;"
        >
          <img
            alt=""
            aria-hidden="true"
            src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iNTAwIiBoZWlnaHQ9IjUwMCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB2ZXJzaW9uPSIxLjEiLz4="
            style="max-width: 100%; display: block; margin: 0px; padding: 0px;"
          />
        </div>
        <noscript />
        <img
          alt="foo"
          data-nimg="true"
          decoding="async"
          src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
          style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%; object-fit: contain;"
        />
      </div>
    </div>
    <h2
      class="text-4xl font-modo font-bold my-8 md:px-8"
    >
      foo
    </h2>
    <p
      class="px-10 md:px-8 text-2xl text-center"
    >
      bar
    </p>
    <div
      class="my-8 w-2/3 mx-auto"
    >
      <a
        href="/#"
      >
        <div
          class="p-3 px-4 py-2 text-2xl border-l-8 bg-w-x2 ease-out duration-[2s] cursor-pointer disabled:bg-grey-light disabled:cursor-not-allowed hover:shadow-fill-green text-grey-dark border-green bg-w-x2 bg-white font-mono font-semibold font-semibold font-mono px-16 py-2 my-4"
        >
          Apply Now
        </div>
      </a>
    </div>
  </div>
</DocumentFragment>
`;
